File: Task3/task3.cs
using System;
public class Program
{
    public static void Main(string[] args)
    {
        int numbers = Convert.ToInt32(Console.ReadLine());
        int count = 0;
        while (numbers != 0)
        {
            int perem = Convert.ToInt32(Console.ReadLine());
            while (perem > 0)
            {
                int ost = perem % 10;
                if (ost % 3 == 0)
                {
                    count += ost;
                }
                perem /= 10;
            }
            numbers--;
        }
        Console.WriteLine(count);
    }
}

File: Task3/task3.swift
import Foundation
func main() {
    let numbers = Int(readLine()!)!
    var count = 0
    for _ in 0..<numbers {
        let perem = Int(readLine()!)!
        var temp = perem
        while temp != 0 {
            let ost = temp % 10
            if ost % 3 == 0 {
                count += ost
            }
            temp /= 10
        }
    }
    print(count)
}
main()
